content/en/seachat/getting-started/01-seachat-intro.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Introduction"
+description: "SeaChat Intro and getting started"
+lead: ""
+date: 2024-03-08 10:43:51.069 +0100
+lastmod: 2024-03-08 10:43:51.069 +0100
+weight: 201
+draft: false
+images: []
+aliases:
+  - /en/seachat/seachat-intro/01-introduction/
+  - /seachat/seachat-intro/01-seachat-intro/
+toc: true
+---
+
+## Welcome to SeaChat
+
+Build and launch an AI agent and voice agent that elevates your workflow in under 10 minutes. SeaChat supports more than 30 document types, including PDF, CSV, DOC, PNG, and JPG, allowing you to upload almost unlimited knowledge to train your AI chatbot tailored to your needs with a seamless transition to human agent when needed.
```

Comment:
Build and launch an AI agent that elevates your workflow in under 10 minutes. SeaChat supports more than 30 document types, including PDF, CSV, DOC, PNG, JPG and more, allowing you to upload almost unlimited knowledge to train your AI voice and chat agents tailored to your needs with a seamless transition to human agent when needed.

> Corrected a few grammar mistake. I think we'd eventually make this more succinct but let's keep this for SEO purpose